Sources/SentryCrash/Recording/SentryCrashReport.c

```diff
@@ -1207,12 +1205,12 @@ writeBinaryImage(
 static void
 writeBinaryImages(const SentryCrashReportWriter *const writer, const char *const key)
 {
-    const int imageCount = sentrycrashdl_imageCount();
-
+    const int imageCount = sentrycrashbic_imageCount();
     writer->beginArray(writer, key);
     {
         for (int iImg = 0; iImg < imageCount; iImg++) {
-            writeBinaryImage(writer, NULL, iImg);
+            SentryCrashBinaryImage *image = sentrycrashbic_getBinaryImageCache(iImg);
```

Comment:
`l`: We are getting a cached binary image, not the cache itself. 
```suggestion
            SentryCrashBinaryImage *image = sentrycrashbic_getCachedBinaryImage(iImg);
```